src/main/java/uk/gov/hmcts/darts/annotation/controller/AnnotationController.java

```diff
@@ -27,8 +29,17 @@ public class AnnotationController implements AnnotationsApi {
         securityRoles = {JUDGE},
         globalAccessSecurityRoles = {JUDGE})
     @Override
-    public ResponseEntity<PostAnnotationResponse> annotationsPost(MultipartFile file, Annotation annotation) {
+    public ResponseEntity<PostAnnotationResponse> postAnnotation(MultipartFile file, Annotation annotation) {
         var annotationId = annotationService.process(file, annotation);
         return ResponseEntity.ok(new PostAnnotationResponse(annotationId));
     }
+
+    @Authorisation(
+        contextId = ANY_ENTITY_ID,
+        globalAccessSecurityRoles = {JUDGE, ADMIN})
```

Comment:
probably want to have `securityRoles = {JUDGE},` here also, as judges can be on probation where they dont have global access

Reply:
I did check this with Dave, he said not.  Possible I misunderstood so will get his input on this